src/main/java/edu/nextstep/racingcar/car/Cars.java

```diff
@@ -0,0 +1,29 @@
+package edu.nextstep.racingcar.car;
+
+import edu.nextstep.racingcar.strategy.RandomStrategy;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Cars {
+
+    private List<RacingCar> cars = new ArrayList<>();
```

Comment:
```suggestion
    private final List<RacingCar> cars;
```
이렇게 불변 객체로 관리해주면 어떨까요?